tests/integration/__init__.py

```diff
@@ -332,3 +332,85 @@ def wait_for_down(self):
 
     def wait_for_up(self):
         self.up_event.wait()
+
+
+class BasicKeyspaceUnitTestCase(unittest.TestCase):
+    """
+    This is basic unit test case that can be leveraged to scope a keyspace to a specific unit test.
+    creates a keyspace named after the testclass with a rf of 1
+    """
+    @classmethod
+    def setUpClass(self):
+        self.commonSetUp('1')
+
+    @classmethod
+    def tearDownClass(self):
+        execute_until_pass(self.session, "DROP KEYSPACE {0}".format(self.ksname))
```

Comment:
If we assume we have ksname, I think we should make it a property of this base class. May also consider providing a fname property based on function name for tables/types/etc. These can be lifted from standard.test_metadata classes.